interop/cats/src/test/scala/caliban/interop/cats/CatsInteropSpec.scala

```diff
@@ -25,7 +25,7 @@ object CatsInteropSpec extends ZIOSpecDefault {
       val inner   = RootContext(SecurityContext(true), LogContext("internal-trace-id"))
 
       def main(inner: RootContext)(implicit runtime: Runtime[RootContext]) =
-        Dispatcher[Effect].use { dispatcher =>
+        Dispatcher.sequential[Effect].use { dispatcher =>
```

Comment:
The previous behavior was `parallel`, any reason to change to `sequential`?

Reply:
Not too sure what I was thinking here 🤦 . Changed it to `parallel`.